interop/alts/server/server.go

```diff
@@ -30,15 +31,24 @@ import (
 	testpb "google.golang.org/grpc/interop/grpc_testing"
 )
 
+const (
+	udsAddrPrefix = "unix:"
+)
+
 var (
 	hsAddr     = flag.String("alts_handshaker_service_address", "", "ALTS handshaker gRPC service address")
-	serverAddr = flag.String("server_address", ":8080", "The port on which the server is listening")
+	serverAddr = flag.String("server_address", ":8080", "The address on which the server is listening")
```

Comment:
It's better to explain a little bit about which type and format of address can be accepted here.

Reply:
Done.